CIP-1694/README.md

```diff
@@ -0,0 +1,1124 @@
+No other actions are possible during the bootstrap phase.
+
+The bootstrap phase ends when **either** of the following occurs:
+1. a sufficient amount of stake is delegated to DReps
+2. a given number of epochs has passed (likely to be a number of months after the hard fork)
+
+#### Final safety measure, post bootstrapping
+
+Many people have stated that they believe that the actual voting turnout will not be so large
+as to be a strain on the throughput of the system.
+We also believe that this is likely to be the case, but when the bootstrap phase ends we will
+put one final, temporary safety	measure in place (this will also allow us to justify a low DRep deposit amount).
+
+For values of $X$ and $Y$ that are still to be determined,
+as soon as the bootstrap phase has ended,
+when we calculate the DReps stake distribution for the next epoch boundary,
+we will consider _only_ those DReps that are _either_ in the top $X$-many DReps ranked by stake amount,
+or those DReps that have at least $Y$ Lovelace.
+Every epoch, the value of $X$ will _increase_ and the value of $Y$ will decrease,
+so that eventually $X$ will be effectively infinite and $Y$ will be zero.
+Note that this is only an incentive, and nothing actually stops any DRep from casting their
+vote (though it will not be counted if it does not meet the requirements).
+
+If the community decides at some point that there is indeed a problem with congestion,
+then a hard fork could be enacted that limits the number of DReps in a more restrictive way.
+
+Reasonable numbers for the initial value of $X$ are probably 5,000-10,000.
+Reasonable numbers for the initial value of $Y$ are probably the total number of Lovelace
+divided by the initial value of $X$.
+
+The mechanism should be set to relax at a rate where the restriction is completely eliminated after
+a period of six months to one year.
+
+#### Other Ideas / Open Questions
+
+##### Pledge-weighted SPO voting
+
+The SPO vote could additionally be weighted by each SPO's pledge.
+This would provide a mechanism for allowing those with literal stake in the game to have a stronger vote.
+The weighting should be carefully chosen.
+
+##### Automatic re-delegation of DReps
+
+A DRep could optionally list another DRep credential in their registration certificate.
+Upon retirement, all of the DRep's delegations would be automatically transferred to the
+given DRep credential.  If that DRep had already retired, the delegation would be transfer
+to the 'Abstain' DRep.
+
+##### No DRep registration
+
+Since the DRep registration does not perform any necessary functions,
+the certificates for (de-)registering DReps could be removed. This
+makes the democracy more liquid since it removes some bureaucracy and
+also removes the need for the DRep deposit, at the cost of moving the anchor that is part of the
+DRep registration certificate into the transaction metadata.
+
+##### Reduced deposits for some government actions
```

Comment:
It is hard to not see each of the governance actions as critical and needing of a good enough barrier to prevent folly.

If anything Action 7 for Info can have something as low as 500 ADA.

Everything else must be substantially high(with Constitution Changes and HardForks being the most valuable/costly changes), proposers should ultimately be able to recognise the value of their proposal or acquire sponsors who do. This will also mitigate spam.

Reply:
I disagree. I'm fine with every deposing being 100 Ada or lower. If we make it too large you can argue that only rich folks get to make decisions, and I think that's a reasonable argument. The point is that no-one has to vote on anything: if people post some spam or malicious stuff & everyone ignores it, nothing happens. And even if some people vote on it, you still need the approval of the CC, which you won't get for that kind of stuff.